docs/getting-started/documentation.md

```diff
@@ -49,7 +49,21 @@ The documentation website can be run locally using Docker Compose:
 docker-compose up docs
 ```
 
-Browse to `http://localhost:${DOCS_LOCAL_PORT}` (<http://localhost:8001> by default) to view the docs site.
+The site is served from `http://localhost` at a port dynamically assigned by Docker. To find the exact address, either look in
+the Docker Desktop interface or run:
+
+```bash
+docker ps -f name=docs
+```
+
+The `PORTS` column should look like:
+
+```console
+0.0.0.0:$PORT->8000/tcp
+```
+
+Where `$PORT` is the dynamically assigned `localhost` port. You can navigate in a browser to <http://localhost:$PORT> to view
+the docs site.
```

Comment:
I am not able to build or run the docs Docker container locally:

![image](https://user-images.githubusercontent.com/3673236/135665246-4bcba4cc-c717-4c2b-bd2b-c5f793b21338.png)

Reply:
I'm actually getting this error on the `dev` branch so maybe it's just me?!?!

Reply:
Weird, I'm getting the same error... I can't tell if it's a problem with `mkdocs` or a problem in our markdown content somewhere? 

One note: for `docs` there's actually nothing to build, since it's using the same image as `dev`. That one has to be built first, before `docs` can run/up.

Reply:
Works for me now!